interface-definitions/service-upnp.xml.in

```diff
@@ -196,11 +196,12 @@
                 <properties>
                   <help>The IP to which this rule applies (REQUIRE)</help>
                   <valueHelp>
-                    <format>ipv4</format>
+                    <format>ipv4net</format>
                     <description>The IPv4 to which this rule applies</description>
                   </valueHelp>
                   <constraint>
-                    <validator name="ipv4-address" />
+                    <validator name="ipv4-host" />
+                    <validator name="ip-prefix"/>
```

Comment:
`ip-prefix` also allows IPv6 addresses. If this is the case, you should use ip-host, too and add ipv6 to the completion helper.

Reply:
Done

Reply:
I think all 4 options should be listed in the completion help: ipv4, ipv4net, ipv6, ipv6net as the validator also allows everything. 

ipv4 shows <x.x.x.x>
ipv4net shows <x.x.x.x/x>
ipv6 shows <h:h:h:h:h:h:h:h>
ipv6net shows <h:h:h:h:h:h:h:h/x>

Also the completion help looks different to the user - we should list all available options - its free for us.